packages/build/src/plugins/resolve.js

```diff
@@ -137,6 +145,26 @@ const handleMissingPlugins = async function ({ pluginsOptions, autoPluginsDir, m
   )
 }
 
+const handleIntegrations = async function ({ integrations, autoPluginsDir, mode, logs }) {
+  const toInstall = integrations.filter((integration) => integration.has_build)
+  await installIntegrationPlugins({ integrations: toInstall, autoPluginsDir, mode, logs })
+
+  return await Promise.all(
```

Comment:
I'm curious if there's a reason why you're creating another Promise here?

```suggestion
  return Promise.all(
```

Reply:
This was a copy/paste + rename job, so hangover from the above method again and I didn't clock the promisey promise